tools/ice-devtools/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "ice-devtools",
-  "version": "1.0.0-beta.21",
+  "version": "1.0.0-beta.24",
   "description": "ice 开发者工具",
   "main": "index.js",
   "bin": {
-    "ice-devtools": "bin/index.js",
-    "ice-devtools-init": "bin/init.js",
-    "ice-devtools-add": "bin/add.js",
-    "ice-devtools-start": "bin/dev-server.js",
-    "ice-devtools-generate": "bin/db.js"
+    "ice-devtools": "bin/ice-devtools.js",
+    "ice-devtools-init": "bin/ice-devtools-init.js",
+    "ice-devtools-add": "bin/ice-devtools-add.js",
```

Comment:
这种文件风格是什么？ 之前的不是更简洁么

Reply:
兼容 `Windows`路径的问题

Reply:
有相关的  issue 描述这个问题吗， 看这只是文件名的变更而已。

Reply:
>When .command() is invoked with a description argument, no .action(callback) should be called to handle sub-commands, otherwise there will be an error. This tells commander that you're going to use separate executables for sub-commands, much like git(1) and other popular tools.
The commander will try to search the executables in the directory of the entry script (like ./examples/pm) with the name program-command, like pm-install, pm-search.

> Options can be passed with the call to .command(). Specifying true for opts.noHelp will remove the option from the generated help output. Specifying true for opts.isDefault will run the subcommand if no other subcommand is specified.
 
文档上的说明